WifiART/src/userinterface/ArtMvcView.java

```diff
@@ -58,17 +87,53 @@
     private final JLabel bufferTextLabel = new JLabel("Hex Output LCI/LCR Buffer: ");
     private final JTextArea bufferTextArea = new JTextArea();
 
-    //LCI Tab Labels
-
-    //Z Tab Labels
-
-    //Usage Tab Labels
-
-    //BSSID Tab Labels
-
-    //LCR Tab Labels
-
-    //Map Image Tab Labels
+    // LCI Tab labels and components
```

Comment:
A general comment is this looks like too many components in one class - especially it looks like you are going to add more components in the class. Consider:
- Is it possible to split to separate view classes? For instance each tab might be in its own class.
- Is there a need to create a class to represent a component, instead of relying on raw Swing components? For instance it looks like many components contain both a JLabel and a JTextField. It might be cleaner to create a class for this.

Reply:
On your first bullet point, my understanding is that it is generally difficult to split up the View compared to the Model or Controller. If I was to split it up, it would only be to create wrapper classes around the functions, rather than having the components reside in several classes. 
On your second bullet point, this is certainly an interesting idea. One problem that this would raise is the fact that my components are all individually formatted to produce the GUI output that I want, so although the JTextFields all have associated JLabels, the JLabels are not always formatted in the same way relative to the JTextFields. Since combining components like this would only end up deleting one line from the instantiation, the benefits might not be worth sacrificing the readability of the code to someone familiar with the Swing library by instead using locally-defined classes that readers will have to familiarize themselves with.

Reply:
I've been looking into what you brought up in your first bullet point and I now think it is possible to have the View separated into several smaller subview classes responsible for each tab's View. This would be more than just having wrapper classes, as the components used in each tab would reside in their tab's subview. The Controller would have separate subcontrollers for each tab, and these would interact with the subview for that tab. Likewise, the subcontrollers will set values for the submodels representing the data for the respective subelement.

I will open an issue for this change and link to the issue in a subsequent PR where I will make this change in the code structure.